server/src/test/java/org/opensearch/search/approximate/ApproximatePointRangeQueryTests.java

```diff
@@ -1188,4 +1224,146 @@ public void testApproximateWithSort() {
             assertTrue("Should approximate with single sort on same field", query.canApproximate(mockContext));
         }
     }
+
+    public void testApproximateRangeWithSearchAfterAsc() throws IOException {
+        if (numericType == NumericType.HALF_FLOAT) {
+            // Skip - HALF_FLOAT uses different fields for storage vs sorting which causes issues with search_after boundary checking during
```

Comment:
This is due to the precision issues during tests with `half_float`. Let me see if I can fix the tests. I 1st want to ensure all the Gradle check is green with this change.